recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "libtiledb-sql" %}
-{% set version = "0.28.0" %}
-{% set sha256 = "6cc2ef3c6f50571a8bede020d59f2cc51aafef6f957b71ea61d15b77ce161422" %}
+{% set version = "0.29.0" %}
+{% set sha256 = "0b3f7b33a9b516b07d830a213193c3a242694c0540e9f1cd27d0bc2ec5c4ef46" %}
```

Comment:
Also need to change https://github.com/conda-forge/libtiledb-sql-feedstock/blob/594af300966765f1e2588d7da2dea7632c772136/recipe/meta.yaml#L49